app/assets/stylesheets/v3/layout_components/footer.scss

```
@@ -52,35 +52,187 @@ body {
 
 // Footer styles
 .page-footer {
-  background: $brand-blue-light;
+  background: $bg-grey-dark;
   position: relative;
+  font-family: 'Lato', sans-serif;
+  padding: $base-padding*2 0;
 
   &__container {
     @include grid-row;
-    max-width: 1180px;
-    padding-top: $base-padding*3;
+    @include breakpoint(medium only) {
+      max-width: 704px;
+    }
+  
+    @include breakpoint(large only) {
+      max-width: 992px;
+    }
+    @include breakpoint(xlarge only) {
+      max-width: 1334px;
+    }
+  
+    @include breakpoint(xxlarge up) {
+      max-width: 1440px;
+    }
   }
 
   &__col {
-    @include medium-6;
     @include large-3;
+    display: none;
+
+    @include breakpoint(large up) {
+      max-width: 992px;
+      display: block;
+    }
+  }
+
+  &__col-mobile {
+
+
+    @include breakpoint(large up) {
+      max-width: 992px;
+      display: none;
+    }
+  }
+
+  &__col-right {
+    @include large-2;
   }
 
   &__col-header {
+    font-size: $base-font-size*1.5;
+    font-weight: 900;
```

I think we have a variable you can use for this now. `$font-weight-regular--lato`
```suggestion
    font-weight: $font-weight-regular--lato;
```

---

Fixed!